doc/how-to/db/crud.rst

```diff
@@ -194,367 +95,328 @@ INSERT
+
+    -- Delete a tuple by the primary index --
+    tarantool> bands.index.primary:delete{3}
     ---
-    - [6, 7, 8]
+    - [3, 'Ace of Base', 1987]
     ...
-    tarantool> s:select{}
+    tarantool> bands:select()
     ---
-    - - [9, 10, 11]
-      - [12, 13, 14]
+    - - [1, 'Roxette', 1986]
+      - [2, 'Scorpions', 1965]
     ...
-    tarantool> -- Delete by a unique secondary key: ok --
-    s.index.sk_uniq:delete{10}
+
+    -- Delete a tuple by a unique secondary index --
+    tarantool> bands.index.band:delete{'Scorpions'}
     ---
-    - [9, 10, 11]
+    - [2, 'Scorpions', 1965]
     ...
-    s:select{}
+    tarantool> bands:select()
     ---
-    - - [12, 13, 14]
+    - - [1, 'Roxette', 1986]
     ...
-    tarantool> -- Delete by a non-unique secondary index: error --
-    tarantool> s.index.sk_non_uniq:delete{14}
+
+    -- Try to delete a tuple by a non-unique secondary index --
+    tarantool> bands.index.year:delete(1986)
     ---
     - error: Get() doesn't support partial keys and non-unique indexes
     ...
-    tarantool> s:select{}
-    ---
-    - - [12, 13, 14]
-    ...
-    tarantool> s:truncate()
+    tarantool> bands:select()
     ---
+    - - [1, 'Roxette', 1986]
     ...
 
-The key must be full: ``delete`` cannot work with partial keys.
-
-.. code-block:: tarantoolsession
-
-    tarantool> s2 = box.schema.create_space('test2')
-    ---
-    ...
-    tarantool> pk2 = s2:create_index('pk2', {parts = {{1, 'unsigned'}, {2, 'unsigned'}}})
-    ---
-    ...
-    tarantool> s2:insert{1, 1}
-    ---
-    - [1, 1]
-    ...
-    tarantool> -- Delete by a partial key: error --
-    tarantool> s2:delete{1}
+    -- Try to delete a tuple by a partial key --
+    tarantool> bands.index.band_year:delete('Roxette')
     ---
     - error: Invalid key part count in an exact match (expected 2, got 1)
     ...
-    tarantool> -- Delete by a full key: ok --
-    tarantool> s2:delete{1, 1}
+
+    -- Delete a tuple by a full key --
+    tarantool> bands.index.band_year:delete{'Roxette', 1986}
     ---
-    - [1, 1]
+    - [1, 'Roxette', 1986]
     ...
-    tarantool> s2:select{}
+    tarantool> bands:select()
     ---
     - []
     ...
-    tarantool> s2:drop()
+
+    -- Delete all tuples --
+    tarantool> bands:truncate()
     ---
     ...
 
+
 .. _box_space-operations-update:
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 UPDATE
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Similarly to ``delete``, ``update`` accepts a full key of any unique index,
-and also the operations to execute.
+:ref:`space_object.update <box_space-update>` allows you to update a tuple identified by the primary key.
+Its :ref:`index_object.update <box_index-update>` counterpart updates a tuple identified by the specified unique index.
 
-``space:update`` is an alias for "update by primary key".
+Similarly to ``delete``, the ``update`` method accepts a full key and an operation to execute.
```

Comment:
> Nothing about the operation in the delete section.

Do you mean the `delete` section should mention `update`?

> Also, It would be nice to have operations other than "=" in the examples.

I did this intentionally and added links to reference docs where more advanced operations are demonstrated.

Reply:
ooh, I see. Will fix